.github/workflows/stale.yaml

```diff
@@ -0,0 +1,26 @@
+name: 'Stale issue handler'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
nit: maybe move the schedule to something like 18:00:00 UTC? something day time and the oncall can check the stale issues during the work hours.